master/buildbot/process/builder.py

```diff
@@ -673,7 +673,7 @@ def getMergeRequestsFn(self):
 
         # then translate False and True properly
         if mergeRequests_fn is False:
-            mergeRequests_fn = None
+            mergeRequests_fn = Builder._skipMergeRequestFn
```

Comment:
@kkateq this comes from buildbot and as I understand the code is intended to pass also a function with a custom logic to check if buildrequest should get merged